lib/config/default.js

```diff
@@ -139,6 +139,16 @@ exports.config = () => ({
    */
   allow_all_headers: false,
 
+  /**
+   * If the data compression threshold is reached in the payload, the
+   * agent compresses data, using gzip compression by default. The
+   * config option `compression_content_encoding` can be set to 'deflate'
+   * to use deflate compression.
+   *
+   * @env NEW_RELIC_COMPRESSED_CONTENT_ENCODING
+   */
+  compressed_content_encoding: 'gzip',
```

Comment:
can you update this to indicate the new default? https://source.datanerd.us/agents/agent-specs/blob/master/Agent-Endpoints.md#agent-behavior

Reply:
also did you open a PR to add this configuration in the docs?

Reply:
Done here: https://github.com/newrelic/docs-website/pull/8888#pullrequestreview-1076532168